packages/apputils-extension/src/shortcuts.tsx

```diff
@@ -62,19 +62,21 @@ export function displayShortcuts(options: IOptions) {
 
   function formatKeys(keys: string[]): JSX.Element {
     const topContainer: JSX.Element[] = [];
-    for (const key of keys) {
+    keys.forEach((key, index) => {
       const container: JSX.Element[] = [];
       for (const ch of key.split(' ')) {
         container.push(
-          <span className={SHORTCUT_KEY_CLASS}>
+          <span className={SHORTCUT_KEY_CLASS} key={ch}>
```

Comment:
`ch` is not a unique key because we have keybindings composed of pressing the same key twice in short succession, e.g. delete cell is <kbd>D</kbd> + <kbd>D</kbd>:

![Screenshot from 2023-05-20 10-17-47](https://github.com/jupyterlab/jupyterlab/assets/5832902/ec605385-c012-4154-9192-72f054a4957e)

maybe another `forEach` here?